schema/commands.yaml

```diff
@@ -29,7 +29,6 @@ COMMANDS:
     - '<CORTX_PATH>/motr/conf/setup.yaml'
   hare:
     - '<CORTX_PATH>/hare/conf/setup.yaml'
-    - '<CORTX_PATH>/hare/conf/setup-ha.yaml'
```

Comment:
what is the path for HA support bundle?

Reply:
Below is the one already present in commands.yaml
 ha:
    - '/opt/seagate/cortx/ha/conf/setup.yaml'
 
 earlier HA  path was under HARE but it has separated now from hare and listed under HA key. 
 @kupranay